src/styles/globals.css

```
@@ -586,3 +642,192 @@
+
+/* Discover: hero slider shown only in AMOLED */
+[data-theme='amoled-strix'] .discover-hero-slider {
+  display: block;
+}
+
+/* CollectionDetails / IssueDetails: darker gradient overlay */
+[data-theme='amoled-strix'] .collection-backdrop-overlay,
+[data-theme='amoled-strix'] .issue-backdrop-overlay {
+  --tw-gradient-from: rgba(0, 0, 0, 0.6);
+  --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to);
+  background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.6), theme('colors.gray.900'));
```

Biome (2.4.6)

[error] Unexpected unknown function: theme

(lint/correctness/noUnknownFunction)